packages/cli/index.ts

```diff
@@ -15,7 +15,17 @@ program.command('serve')
   .option('-h, --host <host>', 'Host to bind to (default: "0.0.0.0")')
   .addOption(variable)
   .option('--trust-proxy [proxy]', 'Trust the X-Forwarded-Host header')
-  .action(serve)
+  .option('--watch', 'Enable watching for changes')
+  .option('--no-watch', 'Disable watching for changes')
+  .action((options) => {
+    (function serve() {
+      const proc = fork(new URL('./lib/command/serve.js', import.meta.url))
```

Comment:
maybe it's worth adding a comment here as a reminder of why we need this hackish fork?